content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Data Services Cloud Console
+version: 0.1.1
+description: Easy to read guides, tips and documentation to rapidly help any
+  developer working with Data Services Cloud Console APIs.
+image: /img/platforms/image-1-dscc.svg
+width: large
+priority: 1
+tags:
+  - data-services-cloud-console
+  - greenlake-common-cloud
+---
+Today, every organization are required to unleash the power of data to drive digital transformation. But fragmented data management tools, manual processes, and infrastructure silos spanning edge to cloud are getting in the way. Customers across industries are struggling with the complexity of managing data and infrastructure, because it  creates a roadblock to innovation and agility, but also drives business risk. And it’s only getting harder as data continues to grow, apps continue to evolve, and infrastructure continues to spread edge-to-cloud.
+
+![Intelligent Data Platform](/img/dscc-high-level-diagram.jpg "DSCC Diagram")
+
+This API provides resources for any customers that are looking to enhance their infrastructure management and data-ops using the programmatical extensions from Data Services Cloud Console. Data Services Cloud Console brings the cloud experience wherever data lives and streamlines data management across your hybrid cloud. It provides a suite of cloud services across your edge, core, and cloud to accelerate data, agility, and innovation for everyone, from data managers to data innovators.
```

Comment:
Since this is the first time you are introducing the API, spell it out in its entirety as opposed to using the pronoun "This". For instance, The Data Service Cloud Console REST API provides resources for..."

Also, you use DSCC as an acronym later. Make sure that you place "(DSCC)" after its first mention so that it is defined for the reader.

Reply:
Agree, here is the paragraph looks like now:

Data Services Cloud Console public REST API provides resources for any customers that are looking to enhance their infrastructure management and data-ops using the programmatical extensions from Data Services Cloud Console. Data Services Cloud Console (DSCC) brings the cloud experience wherever data lives and streamlines data management across your hybrid cloud. It provides a suite of cloud services across your edge, core, and cloud to accelerate data, agility, and innovation for everyone, from data managers to data innovators.